AutomatedTesting/Gem/PythonTests/scripting/VariableManager_UnpinVariableType_Works.py

```diff
@@ -37,81 +49,69 @@ def VariableManager_UnpinVariableType_Works():
     :return: None
     """
 
-    from PySide2 import QtWidgets
-    import azlmbr.legacy.general as general
-
-    import pyside_utils
-    from utils import TestHelper as helper
-    from utils import Report
-    from PySide2.QtCore import Qt
-
-    GENERAL_WAIT = 1.0  # seconds
-
-    def find_pane(window, pane_name):
-        return window.findChild(QtWidgets.QDockWidget, pane_name)
-
-    def click_menu_option(window, option_text):
+    def click_menu_option(self, window, option_text):
         action = pyside_utils.find_child_by_pattern(window, {"text": option_text, "type": QtWidgets.QAction})
         action.trigger()
```

Comment:
This looks like it would be nice to be added to the utils for the variable manager.

Reply:
agreed that this is a handy function to have in the tools file. moved it over and updated the test script.